src/otx/core/data/module.py

```diff
@@ -95,7 +92,7 @@ def __init__(
         self.device = device
 
         self.subsets: dict[str, OTXDataset] = {}
-        self.save_hyperparameters()
+        self.save_hyperparameters(ignore=["input_size"])
```

Comment:
First of all, the reason why it's added is because lightning compares the value if same attribute exists in both model and dataset side. It's ok for other tasks but for anomaly tasks, it makes some errors because 3 parent classes of anomaly model have input_size attribute and it makes tricky problem currently.
And I think we don't have to save this value in `OTXDatamodule` because input_size values are saved in subset config and transforms as below
![image](https://github.com/user-attachments/assets/275791cd-ab62-4e1d-be42-618653b63bda)